client/src/components/Review.jsx

```diff
@@ -30,7 +30,13 @@ class Review extends React.Component {
           <a className={styles.lightGreyPad}> | </a><a className={styles.orange}><b>{isVerified}</b></a>
           <p className={styles.text}>{review.text}</p>
           <p className={styles.grey}>{review.numHelpful} people found this helpful</p>
-          <div className={styles.actionContainer}><button className={styles.buttonFlex}>Helpful</button><a className={styles.lightGreyFlex}>|</a><a className={styles.greyFlex}>Comment</a><a className={styles.lightGreyFlex}>|</a><a className={styles.greyFlex}>Report Abuse</a></div>
+          <div className={styles.actionContainer}>
```

Comment:
airbnb style guide says to use variables deconstructed from the styles object

Reply:
Thanks, I'll make these changes next time I'm improving the client